islandora_entities.module

```diff
@@ -612,7 +612,7 @@ function islandora_entities_citation_access($object, $type) {
       $simplexml = simplexml_load_string($mads);
       $identifiers = $simplexml->identifier;
       foreach ($identifiers as $identifier) {
-        if ($identifier['type'] == 'u1') {
+        if ($identifier['type'] == 'u1' && (string) $identifier) {
```

Comment:
Could we use a function here that explicitly returns a boolean so we can be clear what we're looking for here, e.g., `!empty((string) $identifier)`? I'm assuming we're just making sure it's not an empty string, but it'd be nice to be clear due to PHP truthiness nonsense